synapse/handlers/federation_event.py

```diff
@@ -515,6 +515,7 @@ async def update_state_for_partial_state_event(
                 )
                 return
```

Comment:
Is there logic somewhere to re-attempt to de-partial-state successor events?

Reply:
yeah, [_sync_partial_state_room](https://github.com/matrix-org/synapse/blob/develop/synapse/handlers/federation.py#L1388) (which calls this method) basically works in a loop, finding remaining partial-stated events.